public/js/reducers/chartsReducer.js

```diff
@@ -0,0 +1,97 @@
+const startVals = [{
+    data: [
+        {x: 1, y: 5},
+        {x: 2, y: 5},
+        {x: 3, y: 5},
+        {x: 4, y: 5},
+        {x: 5, y: 5},
+        {x: 6, y: 5},
+        {x: 7, y: 5}
+    ]
+}];
+
+const startAreaVals = [
+    {
+        data: [
+            {x: 1, y: 0},
+            {x: 2, y: 0},
+            {x: 3, y: 0},
+            {x: 4, y: 0},
+            {x: 5, y: 0},
+            {x: 6, y: 0},
+            {x: 7, y: 0}
+        ]
+    }, {
+        data: [
+            {x: 1, y: 100},
+            {x: 2, y: 100},
+            {x: 3, y: 100},
+            {x: 4, y: 100},
+            {x: 5, y: 100},
+            {x: 6, y: 100},
+            {x: 7, y: 100}
+        ]
+    }
+];
+
+// Creates a list of all y values from the x,y value pairs in a series' dataset
+
+const createPartialsList = (series) => {
```

Comment:
Cool yeah I would agree that it makes sense to keep the code where it is for now